tests/profile_service/index.spec.js

```diff
@@ -66,16 +65,17 @@ describe('profileService', () => {
       it('should get an empty receipt from an empty profile share', (done) => {
         getReceipt('a-test-token', privateKeyFile, 'stub-app-id')
           .then((receipt) => {
```

Comment:
Since we are refactoring here, can we change this to `async` instead of using `done()`?

Reply:
Yes... but not now!
We ought to revisit all tests, probably also using `eslint/jest:recommended` which would flag this amongst other things.